src/components/common/CodeReview.js

```diff
@@ -378,6 +391,44 @@ class CodeReview extends Component {
     });
   }
 
+  handleQuickResponse = (parentComment, option) => {
+    const {
+      dispatchPostAppInstanceResource,
+      dispatchPostAction,
+      codeId,
+      userId,
+      isTeacherView,
+    } = this.props;
+    const { line } = parentComment.data;
+    // this.handleAddComment(lineNum, parentId);
+    // this.handleSubmit(NEW_COMMENT_ID, option)
```

Comment:
Maybe say why you're commenting out these lines?

Reply:
Those lines slipped from me during commit-time inspection 😅 I will remove them in the next commit